<launch>
	<node name="ground_station" pkg="ground_station" type="groundstation" respawn="false" output="screen">
	
		<!-- General window parameters ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
		<param name="window_grayscale_color" value="false"/>
		<param name="window_radial_color" value="true"/>
		<param name="telemetry_refresh_rate" value="100"/>
		
		<!-- Altimeter parameters ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
		<param name="altimeter_unit_is_feet" value="false"/>	
		<param name="altimeter_step_value" value="1"/>
		
		<!-- Variometer parameters ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
		<param name="variometer_step_value" value="1000"/>
				
	   <!-- Gauge1 parameters ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
		<param name="gauge1_name" value="Battery Voltage"/>	
		<param name="gauge1_unit" value="V"/>	
		<param name="gauge1_start_value" value="0"/>
		<param name="gauge1_end_value" value="14"/>
		<param name="gauge1_initial_step" value="2"/>
		<param name="gauge1_sub_step" value="0_5"/>
		<param name="gauge1_drawing_step" value="2"/>
		<param name="gauge1_color_strip_order" value="RYG"/>	
		<param name="gauge1_green_strip_start" value="10"/>
		<param name="gauge1_yellow_strip_start" value="8"/>
		<param name="gauge1_red_strip_start" value="0"/>
		
	</node>
</launch>
